system/Autoloader/FileLocator.php

```diff
@@ -90,10 +90,10 @@ public function locateFile(string $file, string $folder = null, string $ext = 'p
 			$file = substr($file, strlen($folder . '/'));
 		}
 
-		// Is not namespaced? Try the application folder.
+		// Is not namespaced? assume a Config class
 		if (strpos($file, '\\') === false)
 		{
-			return $this->legacyLocate($file, $folder);
+			return $this->autoloader->loadClass($file);
```

Comment:
the `Autoloader::loadClass()` actually do require_once file, not sure if it is ok or we can just check is_file Config class here.